### PR TITLE
add option 'single' to the format options for cleaner 1x1 outputs

### PR DESCRIPTION
**Pull request by Vialeon**

### CODE REVIEW

File: pkg/gitqlite/display.go [hunk 1]
```diff
@@ -37,6 +43,35 @@ func DisplayDB(rows *sql.Rows, w io.Writer, format string) error {
...
+	}
+
+	r := make([]string, len(columns))
+	for i, c := range container {
+		if c.Valid {
+			r[i] = c.String
+		}
+	}
+
+	fmt.Printf("%s : %s\n", columns[0], r[0])
```
Comment (patrickdevivo) [inline pkg/gitqlite/display.go:71]: let's have this just print out the value and nothing else:

```go
fmt.Printf("%s\n", r[0])
```

as the main purpose of this format option is to allow easier piping of single value responses into other commands

File: pkg/gitqlite/display.go [hunk 2]
```diff
@@ -126,8 +161,9 @@ func tableDisplay(rows *sql.Rows, write io.Writer) error {
 	}
 	table := tablewriter.NewWriter(write)
 	table.SetHeader(columns)
+	numRows := 0
 	for rows.Next() {
-
+		numRows++
```
Comment (patrickdevivo) [inline pkg/gitqlite/display.go:166]: curious about `numRows` here - is it used anywhere? possibly holdover from testing/outputting something different?

**Review by patrickdevivo — CHANGES_REQUESTED** [review]
looks good! One more comment beyond the ones below - should also add to the README and CLI help text. I can open a PR for that into this branch

File: pkg/gitqlite/display.go (hunk 2 above)
Comment (Vialeon) [reply pkg/gitqlite/display.go:166]: that is a holdover. I was initially thinking of having it be an automatic process but then realized that would involve creating a counter in each of the outputs which would then be an issue if the format had already been specified. Definitely can rm this

**Review by patrickdevivo — APPROVED** [review]

**Pull request merged**